database/mysql.go

```diff
@@ -38,6 +38,17 @@ var (
 		Help: "How many MySQL connections and what status they're in.",
 	}, []string{"state"})
 
+	mysqlConnectionsCounters = kitprom.NewGaugeFrom(stdprom.GaugeOpts{
+		Name: "mysql_connections_counters",
```

Comment:
Shouldn't this be `_gauges` ?

Reply:
I was going off the naming much like how the status one is going off of it. 
These are counter under the hood but the API for prometheus go doesn't allow me to _set_ a value on a counter.

Reply:
Right, gauges record the value at lots of points in time. Counters only increment. https://prometheus.io/docs/concepts/metric_types/#gauge

Prometheus namings try and refer some type-level information about the data. `_seconds` or `_count`

Reply:
yeah hence the counter metric is labelled `wait_ms`

Reply:
```			wait_count: The total number of connections waited for.
			wait_duration: The total time blocked waiting for a new connection.
			max_idle_closed: The total number of connections closed due to SetMaxIdleConns.
			max_idle_time_closed: The total number of connections closed due to SetConnMaxIdleTime.
			max_lifetime_closed: The total number of connections closed due to SetConnMaxLifetime.```